# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: js/xpconnect/loader/ChromeScriptLoader.cpp
# Commit: ac548c2a725b
# Full Hash: ac548c2a725b66db57c81bb24accca2bbe075931
# Author: Olli Pettay <Olli.Pettay@helsinki.fi>
# Date: 2019-07-29 21:38:39
# Description:
#   Bug 1565930, try to make Promise less error prone to compartment mismatches, r=bzbarsky
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D38762
# ==============================================================================

diff -r c4980c083852 -r ac548c2a725b js/xpconnect/loader/ChromeScriptLoader.cpp
--- a/js/xpconnect/loader/ChromeScriptLoader.cpp	Mon Jul 29 08:00:35 2019 +0000
+++ b/js/xpconnect/loader/ChromeScriptLoader.cpp	Mon Jul 29 14:43:54 2019 +0000
@@ -191,7 +191,7 @@
   if (JS_GetPendingException(aCx, &value)) {
     JS_ClearPendingException(aCx);
   }
-  mPromise->MaybeReject(aCx, value);
+  mPromise->MaybeReject(value);
 }
 
 void AsyncScriptCompiler::Reject(JSContext* aCx, const char* aMsg) {